test/one/profiler/test/Test.java

```diff
@@ -41,4 +41,7 @@
     int[] jvmVer() default {};
 
     boolean enabled() default true;
+
+    // Optional inputs to the test method.
+    double[] inputs() default {};
```

Comment:
`double[]` is probably not the best option for a generic case. `String[]` would likely be a better universal choice, similarly to `main` method accepting `String[] args`.
Optionally, the framework could automatically translate inputs to proper types derived from a method signature. This may unnecessarily complicate the code, though.

Reply:
Will accept `String[]` and convert to `double[]` now, leaving the option to automatically support other types.